homeassistant/components/mqtt/client.py

```diff
@@ -99,7 +99,7 @@
 TIMEOUT_ACK = 10
 RECONNECT_INTERVAL_SECONDS = 10
 
-type SocketType = socket.socket | ssl.SSLSocket | Any
+type SocketType = socket.socket | ssl.SSLSocket | Any | "mqtt.WebsocketWrapper"
```

Comment:
I think you don't need the quotes anymore, and may need an different rc/patch version of this PR as well as a dev version

Reply:
I think the cleanest is to take out the typing change for the patch and add it in a future PR so we can do a cherry pick.

I will write tests once I have stable power